routes/users.js

```diff
@@ -7,5 +7,5 @@ var router = express.Router();
 
 router.get('/:id',auth , user_controller.get_user_by_id)
 router.put('/:id',auth , user_controller.update_user_by_id)
-
+router.get('/enrollments/:id', auth, user_controller.get_user_enrollments)
```

Comment:
if we are passing user_id we should keep the variable as user_id.also pass it as query params